components/object-table/src/index.js

```diff
@@ -0,0 +1,88 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Table } from 'govuk-frederic';
+
+import { objectHasValueForKeys, keysFromFields, rowsFromObject } from '@govuk-frederic/utils';
+
+
+/**
+ *
+ * ### Usage
+ * 
+ * Simple
+ * ```jsx
+ * const fields = [
+ *   { key: 'one', heading: 'One' },
+ *   { key: 'two', heading: 'Two' },
+ *   { key: 'three', heading: 'Three' },
+ *   { key: 'four', heading: 'Four' },
+ * ];
+ * const object = { one: 'test', two: 'test', three: '', four: null };
+ * const title = ['Heading'];
+ * 
+ * <ObjectTable fields={fields} object={object} title={title}/>;
+ * ```
+ * 
+ * With skipEmptyValues
+ * ```jsx
+ * const fields = [
+ *   { key: 'one', heading: 'One' },
+ *   { key: 'two', heading: 'Two' },
+ *   { key: 'three', heading: 'Three' },
+ *   { key: 'four', heading: 'Four' },
+ * ];
+ * const object = { one: 'test', two: 'test', three: '', four: null };
+ * const title = ['Heading'];
+ * 
+ * <ObjectTable fields={fields} object={object} title={title} skipEmptyValues={false}/>
+ * ```
+ * 
+ * With hideWithNoValues
+ * ```jsx
+ * const fields = [
+ *   { key: 'one', heading: 'One' },
+ *   { key: 'two', heading: 'Two' },
+ * ];
+ * const object = { };
+ * const title = ['Heading'];
+ *
+ * <ObjectTable fields={fields} object={object} title={title} hideWithNoValues />;
+ * ```
+ */
+const ObjectTable = ({
```

Comment:
should maybe have an empty line after this comment block

Reply:
Hmm, don't know if this would affect `api-docs` but there's no blank line here in any other files in Fred or `govuk-react` so perhaps we had better leave as-is for now.